### DISCUSSION TITLE
Calling route from other script php

### DISCUSSION THREAD

**Discussion by Elmatbaisaad** [symfony/symfony#46539] [Q&A] [answered]
Hello everyone
I am working on a project symfony where a have a route the only work that it do is extract data from server and store it in database the probleme that this route can do that only if i call it in the browser So i hade the idea of creating an other container which contain a php script that will call this route i tried this but it didn't work :
![image](https://user-images.githubusercontent.com/101296742/171355903-3b0d11c1-d4eb-4666-9559-10250585e963.png)
How can i do that please ?
Thank you .

**Comment by stof** [comment]
is `localhost:8080` the right way to access your container *from another container* ? I doubt it as Docker does not forward exposed port from one container to another one (and so it will likely not be `localhost` to access the other container).
Note that this has nothing to do with Symfony at all. It is all about how Docker works.

**Reply by Elmatbaisaad** [reply]
thank you for your answer.
yeah i totally agree with you i keep getting this error:
![image](https://user-images.githubusercontent.com/101296742/171357660-1d173886-dba1-4bfb-b9b8-139f70b5ff30.png)
do you have any idea how can i solve that ?

**Comment by Elmatbaisaad** [comment, marked as answer]
I solve it :
i had to replace localhost with the gateway adress
